RELEASES.md

```diff
@@ -22,13 +22,17 @@ Example pre-release versions include `v0.1.0-alpha1`, `v0.1.0-beta2`, `v0.1.0-rc
 
 ## Pre Release Activity
 
-Most e2e-scenarios for cli, K8, and Azure are covered by the ratify e2e tests. Please refer to this [document](test/validation.md) for the current supported and unsupported tests. 
+Most e2e-scenarios for cli, K8s, and Azure are covered by the Ratify e2e tests. Please refer to this [document](test/validation.md) for the current supported and unsupported tests. 
 
 Please perform manual prerelease validations for the unsupported tests list [here](test/validation.md#unsupported-tests)
 
 Validate that the format of the data returned for external data calls has not changed. If it has changed update the version in `httpserver/types.go` to reflect a change in the format and document the update.
 
 Delete all dev images generated since the previous release under the `ratify-dev` and `ratify-crds-dev` packages. Each dev image tag is prefixed with `dev` followed by the date of creation and then the abbreviated 7 character commit SHA (e.g a build generated on March 8, 2023 from main branch with commit SHA `4cf98388ef33c587ef86b82e05cb0f7de2da2ea8` would be tagged `dev.20230308.4cf9838`).
+
+Copy contents from `dev.helmfile.yaml` to `helmfile.yaml` & `dev.high-availability.helmfile.yaml` to `high-availability.helmfile.yaml`. You MUST update/remove values marked by comments in the files.
```

Comment:
Please correct me if I'm wrong. Looks like dev.helmfile and helmfile only diffs in the last few lines related to the image. Why do we need to do the copy instead of just updating `helmfile.yaml` for each release?

Reply:
Yes, if there's no change in the install helm values for the quickstart between releases, then that's the only difference. True value is when a new value/install steps is added between releases that's been merged on main branch. For backwards compat we can't update the actual helmfile. We need to update a different file for main branch.

Reply:
TODO: explain why the copying is necessary

Reply:
added explanation to RELEASE.md